File: conditionals/time_of_day.py
# Zoe Jimenez, Time of Day python
import time

current = time.time()
local_time = time.localtime(current)
hour = local_time.tm_hour



if hour >= 18 and hour <= 22:
    print("Good evening, the moon is coming up!")
elif hour >= 12:
    print("Good afternoon, have you eaten yet?")
elif hour >= 6:
    print("Goodmorning! Rise and shine!")
else:
    print("Go to bed what are you doing up at this time of day??")

File: conditionals/time_of_day.c
// Zoe Jimenez, Time of Day C
#include <stdio.h>
#include <time.h>



int main(void){

    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    
    if(hour >= 18 && hour <= 22){
        printf("Good evening!");
    }else if(hour >= 12){
        printf("Good afternoon, have you eaten anything?");
    }else if(hour >= 6){
        printf("Goodmorning! Time to begin the day!");
    }else{
        printf("Get to bed what are you doing awake at this time?");
    }

    
    return 0;
}